xpath.d.ts

```diff
@@ -1,11 +1,35 @@
 /// <reference lib="dom" />
 
-type SelectedValue = Node | Attr | string | number | boolean;
-interface XPathSelect {
-    (expression: string, node?: Node): Array<SelectedValue>;
-    (expression: string, node: Node, single: true): SelectedValue | undefined;
+export type SelectedValue = Node | string | number | boolean | null;
+
+export interface XPathSelect {
+    (expression: string, node: Node): Array<Node>;
+    (expressed: string, node: Node, single: false): Array<Node>;
```

Comment:
Is there a reason the parameter is called `expressed` here?

Reply:
Typo. It doesn't affect function though. Fixed.